composer.json

```diff
@@ -140,6 +140,7 @@
         "league/uri": "^6.5|^7.0",
         "masterminds/html5": "^2.7.2",
         "monolog/monolog": "^1.25.1|^2",
+        "nikic/php-parser": "^4.16|^5.0",
```

Comment:
Shouldn't this be `^4.13` like in the composer.json of the component?

Reply:
Thanks for the heads up. The dependency should have been bumped in the component's `composer.json` file too thus allowing for the `getParts()` method to be used.